api/v1alpha1/dynakube_types.go

```diff
@@ -27,6 +27,10 @@ type DynaKubeSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Custom PullSecret",order=8,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:io.kubernetes:Secret"}
 	CustomPullSecret string `json:"customPullSecret,omitempty"`
 
+	// Tokens for ActiveGates
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="AG Tokens",order=2,xDescriptors="urn:alm:descriptor:io.kubernetes:Secret"
+	AGTokensSecret string `json:"agTokensSecret,omitempty"`
```

Comment:
Why is this needed? It is not used when assembling the STS.

Reply:
related discussion: https://github.com/Dynatrace/dynatrace-operator/pull/167#discussion_r658550010

Reply:
We don't need it because secrets name can be extended with dynakube object name, so it can be calculated dynamically instead of declare in spec? Let's discuss it in linked discussion.

Reply:
The rationale for having the `Tokens` field is to share the user provided tokens among multiple DynaKube objects. `AGTokensSecret` is a bit different as it is always fetched via Dynatrace API.